src/java/org/jivesoftware/openfire/handler/IQPrivacyHandler.java

```diff
@@ -51,7 +51,7 @@
 
     private IQHandlerInfo info;
     private PrivacyListManager manager = PrivacyListManager.getInstance();
-    private PrivacyListProvider provider = new PrivacyListProvider();
+    private PrivacyListProvider provider = PrivacyListProvider::getInstance();
```

Comment:
:: is not valid in Java. Static methods can be called like any other: `PrivacyListProvider.getInstance()` should do it.